eslint-common.json

```diff
@@ -9,6 +9,7 @@
   "dot-location": ["property"],
   "eol-last": ["always"],
   "eqeqeq": ["always", {"null": "never"}],
+  "for-direction": [],
```

Comment:
> Any reason to favour the build option?

This project predates ESLint's extension mechanism, which was added fairly recently. I'll investigate the possibility of using `extends` here. Thanks for the suggestion!